authors/apps/profiles/renderers.py

```diff
@@ -22,4 +22,12 @@ def render(self, data, media_type=None, renderer_context=None):
         return json.dumps({
             'profile': data
         })
-        
+
```

Comment:
You should add a single line here: PEP8 guidelines

Reply:
noted